include/wx/defs.h

```diff
@@ -505,6 +505,14 @@ typedef short int WXTYPE;
     #endif
 #endif /* HAVE_WOSTREAM */
 
+#ifndef wxHAVE_INITIALIZER_LIST
+    #if __cplusplus >= 201103L
+        #define wxHAVE_INITIALIZER_LIST
+    #elif wxCHECK_VISUALC_VERSION(11)
```

Comment:
Visual Studio 10 offers an early support for C++11, but that does not include `initializer_list`.
Real support for it starts with Visual Studio 11.